체육복.js

```diff
@@ -0,0 +1,31 @@
+function solution(n, lost, reserve) {
+    let answer = 0;
+    let ptArr = [];
+    for (let i = 1 ; i <= n ; i++) {
+        ptArr.push(1) 
+        if (reserve.includes(i)) { 
+            ptArr[i-1]++   
+        } 
+        if (lost.includes(i)) { 
+            ptArr[i-1]-- 
+        }       
+    }
+    for (let i = 0 ; i < n ; i++) {
+        if (!ptArr[i]) { 
+            if (ptArr[i-1]===2) { 
+                ptArr[i]++ 
+                ptArr[i-1]-- 
+        } else if(ptArr[i+1]===2){ 
+            ptArr[i]++ 
+            ptArr[i+1]-- 
+        }
+    }
+    }
+    for (let i = 0 ; i< n ; i ++) { 
+        if (ptArr[i] >= 1) {
+            answer++
+        }
+    }
+
+    return answer;
+}
```

Comment:
1. 배열은 const선언
2. 변수 선언 순서 -> (const > let)
3. 문단마다 줄바꿈 한 줄씩
4. 공백, 들여쓰기 신경쓰기
5. for문이 여러개일 때 (i > j > k) 순으로 선언
5. if문의 실행문이 한 줄일 땐 중괄호 생략 후 한 줄로 조건-실행문 작성
6. 세미콜론 붙이기
```suggestion
function solution(n, lost, reserve) {
    const ptArr = [];
    let answer = 0;
    
    for (let i = 1; i <= n; i++) {
        ptArr.push(1);
        if (reserve.includes(i)) ptArr[i - 1]++;
        if (lost.includes(i)) ptArr[i - 1]--;     
    }
    
    for (let j = 0; j < n; j++) {
        if (!ptArr[j]) {
            if (ptArr[j - 1] === 2) { 
                ptArr[j]++; 
                ptArr[j - 1]--;
            }
            else if (ptArr[j + 1] === 2) {
                ptArr[j]++;
                ptArr[j+1]--; 
            }
        }
    }
    
    for (let k = 0; k < n; k++) { 
        if (ptArr[k] >= 1) answer++;
    }

    return answer;
}
```